### ISSUE TITLE
Added a friend in onboarding, but they weren't part of my circle when I got into the app

### ISSUE THREAD

**Issue by cstiens** [guardianproject/OpenCircle#54]
When I set the app up, I searched for a friend and added them from the onboarding 'add friends' view. Once I got into the app, though, I had no one in my circle.

**Comment by n8fr8** [comment]
Right, because there was no circle yet in the onboarding... they should be in your local friend list however.
